tests/test_count_connected_components.py

```diff
@@ -0,0 +1,40 @@
+#import modules
+import unittest
+from algorithms.graph import count_connected_number_of_component
+
+class TestConnectedComponentInGraph(unittest.TestCase):
+	""" Class to performs unit tests """
+	
+	def test_count_connected_components(self):
+		"""
+		   Test Function that test the different cases of count connected components
+	        
+		    0----------2    1--------5      3    
+	            |
+	            |
+	            4
+	            
+	            output = 3
+
+		"""
+		expected_result = 3
+		
+		# adjacency list representation of graph
```

Comment:
Could you please add more tests? I mean, empty graph as input, or a set of nodes without edges, to increase the test cases coverage.

Reply:
sure! 
on the way